inc/views/base/postbit/postbit.twig

```diff
@@ -0,0 +1,46 @@
+{{ ignore_bit|raw }}{{ deleted_bit|raw }}
+<a name="pid{{ post.pid }}" id="pid{{ post.pid }}"></a>
+<div class="post {{ unapproved_shade }}" style="{{ post_visibility }}" id="post_{{ post.pid }}">
+	<div class="post_author">
+		{{ post.useravatar|raw }}
+		<div class="author_information">
+				<strong><span class="largetext">{{ post.profilelink|raw }}</span></strong> {{ post.onlinestatus|raw }}<br />
+				<span class="smalltext">
+					{{ post.usertitle|raw }}<br />
+					{{ post.userstars|raw }}
+					{{ post.groupimage|raw }}
+				</span>
+		</div>
+		<div class="author_statistics">
+			{{ post.user_details|raw }}
```

Comment:
Might be worth moving the content of the user details tmeplate inline, since the plan is to have one postbit template.

Reply:
Will do.